Description: Do not force custom scheme registration in user's directory
 A user still can call to this method via "registertg" cheat code in Settings.
Discussion: https://github.com/telegramdesktop/tdesktop/pull/8627#issuecomment-720578885
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Sun, 07 Feb 2021 10:55:30 +0300

--- a/Telegram/SourceFiles/core/application.cpp
+++ b/Telegram/SourceFiles/core/application.cpp
@@ -198,7 +198,9 @@ void Application::run() {
 	ValidateScale();
 
 	if (Local::oldSettingsVersion() < AppVersion) {
+#ifndef DESKTOP_APP_USE_PACKAGED
 		RegisterUrlScheme();
+#endif // DESKTOP_APP_USE_PACKAGED
 		psNewVersion();
 	}
 
